vanilla/vanilla
# Add re-orienting of images uploaded with media API

Creating a new image via the media API v2 endpoint does not reorient an image based on its EXIF orientation data. This is addressed here by pre-processing new images with `ImageResizer::resize`, which performs several operations on images, including orientation correction.

### Testing
1. Upload an image with orientation EXIF data through Rich Editor. It should be re-oriented.
1. Enable image dimension limits in the dashboard (under "Posting").
1. Upload the same image again, ensuring its dimensions exceed the limits you set in the previous step. It should be resized and reoriented.

If you need an example image, please let me know.

Closes vanilla/support#409
